DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -11,442 +11,452 @@ CovarianceParameterization pat::PackedCandidate::covarianceParameterization_;
 std::once_flag pat::PackedCandidate::covariance_load_flag;
 
 void pat::PackedCandidate::pack(bool unpackAfterwards) {
-    packedPt_  =  MiniFloatConverter::float32to16(p4_.load()->Pt());
-    packedEta_ =  int16_t(std::round(p4_.load()->Eta()/6.0f*std::numeric_limits<int16_t>::max()));
-    packedPhi_ =  int16_t(std::round(p4_.load()->Phi()/3.2f*std::numeric_limits<int16_t>::max()));
-    packedM_   =  MiniFloatConverter::float32to16(p4_.load()->M());
-    if (unpackAfterwards) {
-      delete p4_.exchange(nullptr);
-      delete p4c_.exchange(nullptr);
-      unpack(); // force the values to match with the packed ones
-    }
+  packedPt_ = MiniFloatConverter::float32to16(p4_.load()->Pt());
+  packedEta_ = int16_t(std::round(p4_.load()->Eta() / 6.0f * std::numeric_limits<int16_t>::max()));
+  packedPhi_ = int16_t(std::round(p4_.load()->Phi() / 3.2f * std::numeric_limits<int16_t>::max()));
+  packedM_ = MiniFloatConverter::float32to16(p4_.load()->M());
+  if (unpackAfterwards) {
+    delete p4_.exchange(nullptr);
+    delete p4c_.exchange(nullptr);
+    unpack();  // force the values to match with the packed ones
+  }
 }
 
 void pat::PackedCandidate::packVtx(bool unpackAfterwards) {
-    reco::VertexRef pvRef = vertexRef();
-    Point pv = pvRef.isNonnull() ? pvRef->position() : Point();
-    float dxPV = vertex_.load()->X() - pv.X(), dyPV = vertex_.load()->Y() - pv.Y(); //, rPV = std::hypot(dxPV, dyPV);
-    float s = std::sin(float(p4_.load()->Phi())+dphi_), c = std::cos(float(p4_.load()->Phi()+dphi_)); // not the fastest option, but we're in reduced precision already, so let's avoid more roundoffs
-    dxy_  = - dxPV * s + dyPV * c;    
-    // if we want to go back to the full x,y,z we need to store also
-    // float dl = dxPV * c + dyPV * s; 
-    // float xRec = - dxy_ * s + dl * c, yRec = dxy_ * c + dl * s;
-    float pzpt = p4_.load()->Pz()/p4_.load()->Pt();
-    dz_ = vertex_.load()->Z() - pv.Z() - (dxPV*c + dyPV*s) * pzpt;
-    packedDxy_ = MiniFloatConverter::float32to16(dxy_*100);
-    packedDz_   = pvRef.isNonnull() ? MiniFloatConverter::float32to16(dz_*100) : int16_t(std::round(dz_/40.f*std::numeric_limits<int16_t>::max()));
-    packedDPhi_ =  int16_t(std::round(dphi_/3.2f*std::numeric_limits<int16_t>::max()));
-    packedDEta_ =  MiniFloatConverter::float32to16(deta_);
-    packedDTrkPt_ = MiniFloatConverter::float32to16(dtrkpt_);
- 
-    if (unpackAfterwards) {
-      delete vertex_.exchange(nullptr);
-      unpackVtx();
-    }
+  reco::VertexRef pvRef = vertexRef();
+  Point pv = pvRef.isNonnull() ? pvRef->position() : Point();
+  float dxPV = vertex_.load()->X() - pv.X(), dyPV = vertex_.load()->Y() - pv.Y();  //, rPV = std::hypot(dxPV, dyPV);
+  float s = std::sin(float(p4_.load()->Phi()) + dphi_),
+        c = std::cos(float(
+            p4_.load()->Phi() +
+            dphi_));  // not the fastest option, but we're in reduced precision already, so let's avoid more roundoffs
```

Comment:
@davidlange6 @fabiocos 
are there some options in clang-format that could make this less broken up?